src/data/grammar.json

```diff
@@ -62,7 +62,7 @@
         "#titleMoney#: #investType# #card# #investReason#",
         "#titleMoney#: #investType# #card#",
         "#titleMoney#: #pro# is #investType#ing #broadCardClassification.capitalizeAll#; Should You?",
-        "BREAKING: Prices of #card.capitalizeAll# Spiking; Market Cornered by #pros#",
```

Comment:
yeah, I prefer the former.

I suppose we could update the coffee-stained rule and its ilk to use capitalizeAll, but we shouldn't do that before we fix capitalizeAll!